packages/ffe-account-selector-react/src/components/account-selector/AccountSelector.js

```diff
@@ -50,9 +50,9 @@ class AccountSelector extends Component {
         this.props.onAccountSelected(account);
     }
 
-    onInputChange(value) {
+    onInputChange(value, event) {
         this.enableFilter = true;
-        this.props.onChange(value);
+        this.props.onChange(value, event);
```

Comment:
Skulle man hatt en test i `AccountSelector.spec.js` som sjekker at den blir kalt med value og event?
`calls onChange with value and event when the input field value changes` eller noe